src/Monolog/Handler/MailHandler.php

```diff
@@ -70,7 +70,7 @@ protected function getHighestRecord(array $records): array
 
     protected function isHtmlBody(string $body): bool
     {
-        return substr($body, 0, 1) === '<';
+        return $body[0] === '<';
```

Comment:
This will break when an empty string is passed.

Reply:
@xabbuh `$body` is always `string` (type hinted), so that can not happen!